### PR TITLE
Remove Lens from production guides

### PR DESCRIPTION
**Pull request by DaruZero**
# Description

This PR aims to remove as much references to Lens as possible. The main job is to just translate whatever action was performed in Lens into a normal shell command, not to adjust the guides themselves (see also additional info below).

I did not remove Lens from past upgrade guides, pages marked as draft (k3d installation) and pages that will be deprecated (see also additional info below).

## Related issues

Close https://github.com/united-manufacturing-hub/MgmtIssues/issues/1322

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

## Additional information

There are many guides that may need a complete revision or even a definitive deprecation. Here's a non-comprehensive list that I wrote while working on this PR:

**To revise**:
- `administration/access-services-from-outside-cluster.md`
- `administration/delete-assets.md`
- `administration/optimize-database-datatypes.md`
- `administration/optimize-time-consuming-queries.md`
- `administration/reduce-database-size.md`

**To deprecate**:
- `administration/access-kafka-outside-cluster.md`
- `security/change-vernemq-acl-configuration.md`

In general though, I would give a deeper look into all the guides, and test if they still work with the latest changes. 

Also, in the [historian ](https://umh.docs.umh.app/docs/datamodel/database/#data-retrieval) page of the new data model there are instruction on how to access the database. Personally, I would suggest to just update the guide on how to access the database and reference it from the data model page

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/production-guide/administration/access-database.md [hunk 1]
```diff
@@ -30,15 +30,16 @@ you just want to see the data in a dashboard and don't need to manupulate it.
 ### Get the database credentials
 
 If you are not using the CLI, you need to know the database credentials. You can
-find them in the **{{< resource type="secret" name="db-psw">}}** Secret. By
-default, the username is factoryinsight and the password is changeme.
+find them in the **{{< resource type="secret" name="db-psw">}}** Secret. Run the
+following command to get the credentials:
 
 ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/access-database.md:36]: This is used I think in different places now. Previously I added an test that checks if this works. Can you modify the test as well that checks if this works? Additionally, can you ensure through comments etc. that any changes here must pass the automated test?

Comment (JeremyTheocharis) [reply umh.docs.umh.app/content/en/docs/production-guide/administration/access-database.md:36]: `<!-- This command is tested within #1153 -->`

File: umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md [hunk 2]
```diff
@@ -63,21 +53,14 @@ The [Kafka Broker](/docs/architecture/microservices/core/kafka-broker/) uses the
...
 Follow these steps to access the Kafka Broker outside the cluster:
 
-1. Open UMHLens / OpenLens and navigate to **Network > Services**.
-2. Select the Service `united-manufacturing-hub-kafka-external` and click the **Edit** button.
-3. Scroll down to the line that shows:
-
-   ```yaml
-     type: NodePort
-   ```
+1. Access your instance via SSh
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md:56]: SSH

File: umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md [hunk 3]
```diff
@@ -63,21 +53,14 @@ The [Kafka Broker](/docs/architecture/microservices/core/kafka-broker/) uses the
...
+2. Execute this command:
 
-   Replace this with:
-
-   ```yaml
-     type: LoadBalancer
+   ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md:59]: Can we add a test that will test whether this command will work? similar to the ones to get credentials?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/execute-kafka-shell-scripts.md [hunk 4]
```diff
@@ -19,27 +19,26 @@ administrative tasks. This page describes how to execute Kafka shell scripts.
 
 ## Open a shell in the Kafka container
 
-1. From the Pod section in {{< resource type="lens" name="name" >}}, click on **{{% resource type="pod" name="kafka" %}}**
-   to open the details page.
-2. {{< include "pod-shell.md" >}}
-3. Navigate to the Kafka bin directory:
+1. From the instance's shell, execute this command:
 
    ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/execute-kafka-shell-scripts.md:24]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/explore-cached-data.md [hunk 5]
```diff
@@ -20,23 +20,31 @@ and explore the data.
...
-1. Open {{< resource type="lens" name="name" >}} and navigate to the **Config** >
-   **Secrets** page.
-2. Get the cache password from the Secret **{{< resource type="secret" name="cache" >}}**.
-3. From the **Pods** section click on **{{% resource type="pod" name="cache" %}}**
-   to open the details page.
+Get access to the instance's shell and execute the following commands.
+
+1. Get the cache password
+
+   ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/explore-cached-data.md:27]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/expose-grafana-to-internet.md [hunk 6]
```diff
@@ -19,19 +19,16 @@ it from outside the Kubernetes cluster.
...
-1. Open {{< resource type="lens" name="name" >}} and go to the **Helm** > **Releases**
-page.
-2. Click the **Upgrade** button and search for Grafana.
-3. Scroll down to the `ingress` section
-4. Set the `enabled` field to `true`.
-5. Add you domain name to the `hosts` field.
-6. Click **Upgrade** to apply the changes.
+To do so, run the following command:
+
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/expose-grafana-to-internet.md:26]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md [hunk 7]
```diff
@@ -20,35 +20,21 @@ with proper security measures.
...
-      securityContext:
-        runAsUser: 0
-        runAsNonRoot: false
-        fsGroup: 0
-   ...
-   ```
+From the instance's shell, execute this command:
 
-5. Click **Save**.
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md:25]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md [hunk 8]
```diff
@@ -20,35 +20,21 @@ with proper security measures.
...
 
 ## Install the packages
 
-1. From the Pods section in {{< resource type="lens" name="name" >}}, click on **{{% resource type="pod" name="nodered" %}}**
-   to open the details page.
-2. {{< include "pod-shell.md" >}}
-3. Install the packages with `apk`:
+1. Open a shell in the {{% resource type="pod" name="nodered" %}} pod with:
+
+      ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md:33]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md [hunk 9]
```diff
@@ -62,34 +48,18 @@ with proper security measures.
...
-      securityContext:
-        runAsUser: 1000
-        runAsNonRoot: true
-        fsGroup: 1000
-   ...
-   ```
+From the instance's shell, execute this command:
 
-4. Click **Save**.
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/install-custom-drivers-nodered.md:60]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/administration/optimize-time-consuming-queries.md [hunk 10]
```diff
@@ -42,13 +48,12 @@ If you have already created an index, you can rollback the factoryinsight deploy
...
-1. From the Deployments section in {{< resource type="lens" name="name" >}}, click
-   on **{{< resource type="deployment" name="factoryinsight" >}}** to open the
-   details page.
-2. {{< include "deployment-edit.md" >}}
-3. Scroll down to the `spec.containers` section and change the `image` value to
-   {{< resource type="docker" name="org" >}}/factoryinsight:0.9.4.
-4. Click **Save**.
+From the instance's shell, run the following command:
+
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/administration/optimize-time-consuming-queries.md:53]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md [hunk 11]
```diff
@@ -98,10 +104,9 @@ This step could take a while depending on the size of the backup file.
 
 ### Temporarly disable kafkatopostrgesql
 
-1. Navigate to **Workloads** > **Deployments**.
-2. Select the {{< resource type="deployment" name="kafkatopostgresql" >}} Deployment.
-3. {{< include "deployment-scale" >}}
-4. Scale the number of replicas to 0.
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md:107]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md [hunk 12]
```diff
@@ -143,10 +148,9 @@ This step could take a while depending on the size of the backup file.
 
 ### Enable kafkatopostgresql
 
-1. Navigate to **Workloads** > **Deployments**.
-2. Select the {{< resource type="deployment" name="kafkatopostgresql" >}} Deployment.
-3. {{< include "deployment-scale" >}}
-4. Scale the number of replicas to the original value, usually 1.
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md:151]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md [hunk 13]
```diff
@@ -59,11 +65,12 @@ Create a Grafana API Token for an admin user by following these steps:
...
 To prevent data inconsistencies, you need to temporarily stop the MQTT and Kafka Brokers.
 
-1. In {{< resource type="lens" name="name" >}} go to the **Workloads** > **StatefulSets** tab.
-2. Select the {{< resource type="statefulset" name="kafkabroker" >}} StatefulSet
-3. {{< include "statefulset-scale.md" >}}
-4. Set the number of replicas to 0 and click **Scale**.
-5. Repeat the process for the {{< resource type="statefulset" name="mqttbroker" >}} StatefulSet.
+Access the instance's shell and execute the following commands:
+
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md:70]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md [hunk 14]
```diff
@@ -16,12 +16,13 @@ aliase:
...
-2. Navigate to **Helm** > **Releases**.
-3. Select the {{< resource type="helm" name="release" >}} release and click **Upgrade**.
-4. Find the `mqtt_broker` section.
-5. Locate the `rbacEnabled` parameter and change its value from `false` to `true`.
-6. Click **Upgrade**.
+Enable RBAC by upgrading the value in the Helm chart.
+
+To do so, run the following command:
+
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md:23]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md [hunk 15]
```diff
@@ -30,17 +31,19 @@ Now all MQTT connections require password authentication with the following defa
 
 ## Change default credentials
 
-1. Open {{< resource type="lens" name="name" >}}
-2. Navigate to **Workloads** > **Pods**.
-3. Select the {{< resource type="pod" name="mqttbroker" >}} Pod.
-4. {{< include "pod-shell" >}}
-5. Navigate to the installation directory of the RBAC extension.
+1. Open a shell inside the Pod:
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md:34]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md [hunk 16]
```diff
@@ -50,26 +53,35 @@ Now all MQTT connections require password authentication with the following defa
...
 
-8. Navigate to **Config** > **ConfigMaps**.
-9. Select the {{< resource type="configmap" name="mqttbroker-credentials" >}} ConfigMap.
-10. Click the **Edit** button to open the ConfigMap editor.
-11. In the `data.credentials.xml` section, replace the strings inbetween the
-    `<password>` tags with the password hash generated in step 7.
+5. Exit the shell by typing `exit`.
+6. Edit the ConfigMap to update the password hash.
+
+    ```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md:65]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md [hunk 17]
```diff
@@ -50,26 +53,35 @@ Now all MQTT connections require password authentication with the following defa
...
     remember that you will need to update the configuration in each one
     to use the new password.
     {{< /notice >}}
-12. Click **Save** to apply the changes.
-13. Go back to **Workloads** > **Pods** and select the {{< resource type="pod" name="mqttbroker" >}} Pod.
-14. {{< include "pod-delete" >}}
+
+7. Save the changes.
+8. Recreate the Pod:
+
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/security/enable-rbac-mqtt-broker.md:81]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/docs/production-guide/security/setup-pki-mqtt-broker.md [hunk 18]
```diff
@@ -127,23 +127,33 @@ You could also do it manually with the following command:
...
+
+Now, send copy it to your instance with the following command:
+
+```bash
+scp pki.yaml <username>@<ip-address>:/tmp
+```
+
+After that, access the instance with SSH and run the following command:
+
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/security/setup-pki-mqtt-broker.md:154]: Can we add a test checking that this command will work in the E2E tests?

File: umh.docs.umh.app/content/en/includes/open-database-shell.md [hunk 19]
```diff
@@ -1,14 +1,5 @@
-1. From the Pod section in {{< resource type="lens" name="name" >}}, click on **{{% resource type="pod" name="database" %}}**
-   to open the details page.
-2. {{< include "pod-shell.md" >}}
-3. Enter the postgres shell:
+```bash
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/includes/open-database-shell.md:1]: Can we add a test checking that this command will work in the E2E tests?

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Comment by DaruZero** [comment]
Added tests for most of the commands in [this PR](https://github.com/united-manufacturing-hub/ManagementConsole/pull/258). `helm upgrade` commands are quite tricky to test, thus I left them out of the PR. Same goes for accessing the `psql` shell in the TimescaleDB pod. Everything else is tested

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**